alchy/model.py

```diff
@@ -126,12 +125,7 @@ class User(Model):
     @declared_attr
     def __table_args__(cls):  # pylint: disable=no-self-argument
         # pylint: disable=no-member
-        args, kargs = merge_declarative_args(cls.__bases__, '__table_args__')
```

Comment:
Previously, each `__table_args__` from all class bases were merged with `__local_table_args__`. Is that no longer what's happening here? It seems like only the `__local_table_args__` are being merged together and any `__table_args__` are discarded. Is that your intention? Or am I reading this incorrectly?